tier/plural/notes.tpl

```diff
@@ -1 +1,5 @@
 Your tier installation is available at https://{{ .Values.hostname }}
+
+You can inspect the runtime configuration by running:
+
+  curl -u "{{ .Values.tier.basicAuth }}:" https://{{ .Values.hostname }}/v1/whoami
```

Comment:
isn't the basic auth here: `-u "auth:{{ .Values.tier.basicAuth }}"` or am I misunderstanding

Reply:
In your example, `auth` is the user and `basicAuth` would be the password, but I'm using `basicAuth` as the user without a password. It's a common use case when you only want to simulate an API key.